code/modules/materials/material_recipes.dm

```diff
@@ -54,19 +54,23 @@
 	if(integrity >= 50 && hardness >= MAT_VALUE_FLEXIBLE + 10)
 		. += new/datum/stack_recipe/furniture/door(src)
 		. += new/datum/stack_recipe/furniture/barricade(src)
+		. += new/datum/stack_recipe/furniture/barricade/spikes(src)
```

Comment:
Spikes are added to the barricade after construction - doesn't need a separate recipe imo.

Reply:
Ooooooooh, I actually didn't ever realized this was a thing...